spec/bolt/module_installer/puppetfile/git_module_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/module_installer/puppetfile/git_module'
+
+describe Bolt::ModuleInstaller::Puppetfile::GitModule do
+  let(:name) { 'yaml' }
+  let(:git)  { 'https://github.com/puppetlabs/puppetlabs-yaml' }
+  let(:ref)  { '79f98ffd3faf8d3badb1084a676e5fc1cbac464e' }
+  let(:mod)  { described_class.new(name, git, ref) }
+
+  context '#to_spec' do
+    it 'returns a Puppetfile spec' do
```

Comment:
Haven't finished reviewing yet, but we should have a test that also verifies that this line gets added to a Puppetfile and then that puppetfile installs correctly if we can.

Reply:
Added as part of the integration test in cli_spec